UPGRADE.md

```diff
@@ -1,3 +1,10 @@
+# Upgrade to 2.6
```

Comment:
Removing it is a BC break, so removing it would force to go to 3.0 (or to stop following semver, which would be a pain for all Composer users)